### PR TITLE
elasticmq-server: 0.14.6 -> 1.2.0, add passthru.tests

### PR DESCRIPTION
**Pull request by risicle**
###### Motivation for this change
Added quite a simple test to check basic operation of the queue.

Still, it's the first `passthru.tests` I've written, and I don't know whether e.g. detaching a process here is considered "ok". It feels a little less wrong than doing it in `checkPhase`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
@ofborg build elasticmq-server-bin elasticmq-server-bin.tests

**Comment by SuperSandro2000** [comment]
@risicle please resolve the merge conflict.

**Comment by risicle** [comment]
@ofborg build elasticmq-server-bin elasticmq-server-bin.tests

File: pkgs/servers/elasticmq-server-bin/default.nix [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
+      homepage = "https://github.com/softwaremill/elasticmq";
+      description = "Message queueing system with Java, Scala and Amazon SQS-compatible interfaces";
+      license = licenses.asl20;
+      platforms = platforms.unix;
+      maintainers = with maintainers; [ peterromfeldhk ];
+    };
+  };
+in elasticmq-server.overrideAttrs (_: {
+  passthru.tests.elasticmqTest = import ./elasticmq-test.nix {
+    inherit elasticmq-server runCommand python3Packages writeText;
```
Comment (doronbehar) [inline pkgs/servers/elasticmq-server-bin/default.nix:38]: Why not add it as is, without the whole `let in` thing.

Comment (risicle) [reply pkgs/servers/elasticmq-server-bin/default.nix:38]: What reference to the `elasticmq-server` package would it use in that case?

**Comment by doronbehar** [comment]
Result of `nixpkgs-review pr 97299` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>elasticmq-server-bin</li>
  </ul>
</details>

**Review by doronbehar — APPROVED** [review]

**Comment by doronbehar** [comment]
@risicle perhaps now we should update to https://github.com/softwaremill/elasticmq/releases/tag/v1.2.0 ?

**Comment by risicle** [comment]
Totally forgot about this. Yes, perhaps.

**Comment by risicle** [comment]
Rebased, bumped, tests pass on nixos x86_64 and macos 10.15

**Review by doronbehar — APPROVED** [review]

**Pull request merged**